src/pages/page-names.constants.ts

```diff
@@ -53,7 +53,7 @@ export const CAT_C = {
   DEBRIEF_PAGE: 'DebriefCatCPage',
   HEALTH_DECLARATION_PAGE: 'HealthDeclarationCatCPage',
   OFFICE_PAGE: 'OfficeCatCPage',
-  PASS_FINALISATION_PAGE: 'PassFinalisationCatCPage',
+  PASS_FINALISATION_PAGE: 'PassFinalisationCatDPage',
```

Comment:
Don't think you meant to change this one :)

Reply:
Yeah, this would break the app

Reply:
done